lib/jira/client/axios.js

```diff
@@ -1,86 +1,61 @@
 const axios = require('axios')
 const jwt = require('atlassian-jwt')
+const querystring = require('querystring')
 const url = require('url')
 
 const { logger } = require('probot/lib/logger')
 
 const instance = process.env.INSTANCE_NAME
 const iss = 'com.github.integration' + (instance ? `.${instance}` : '')
 
+/*
+ * Create an Atlassian request from an Axios config.
+ * Compatible with jwt.createQueryStringHash.
+ */
+jwt.reqFromAxiosConfig = (config) => {
+  const parsedUrl = url.parse(config.url)
+  const parsedQuery = querystring.parse(parsedUrl.query)
```

Comment:
Do we need to parse the query string in the first place? If not, I would opt for backward compatibility and just not doing it at all.
If we need to, 👍